lesson1/src/stacks_queues/stackProblems.js

```diff
@@ -21,9 +21,12 @@ class StackProblems {
     let stack = new Stack();
     for (let i = 0; i < string.length; i++) {
       let char = string.charAt(i);
-      // todo: do you'r magic ! 🙌
+      if (char == '(') {
```

Comment:
this is the general idea and you are definitely on the right path! , but you are missing some more cases, what happens if the string only contains one closing parenthesis? stack will pop (will not fail in javascript) and stack.isEmpty will return true and a string with only one closing parenthesis is certainly not balanced!